src/sentry/integrations/github/client.py

```diff
@@ -709,10 +710,11 @@ def get_blame_for_files(self, files: Sequence[SourceLineInfo]) -> Sequence[FileB
         file_path_mapping = generate_file_path_mapping(files)
 
         try:
-            response = self.post(
+            response = self.post_cached(
                 path="/graphql",
                 data={"query": create_blame_query(file_path_mapping)},
```

Comment:
Does `data` get used in the cache key? Looking at the `_get_cached()` method it looks like it's only using `path` and `params`.

Reply:
I don't think so, it's just the integration name and some hash